cogs/urbandictionary_cog.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+The Urban Dictionary cog written by Ben Ridings,
+You can randomly search a word or search a specific word
+"""
+import json
+
+import aiohttp
+from discord.ext import commands
+
+# Constantы
```

Comment:
Remove the Russian character from the comment, and make the comment a bit more helpful for readers